symbology/dictionary-symbol-sample/src/main/java/com/esri/samples/symbology/SymbolDictionarySample.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2015 Esri. Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law
+ * or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+package com.esri.samples.symbology;
+
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.geometry.Insets;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Control;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.symbology.CimSymbol;
+import com.esri.arcgisruntime.symbology.StyleSymbolSearchParameters;
+import com.esri.arcgisruntime.symbology.StyleSymbolSearchResult;
+import com.esri.arcgisruntime.symbology.SymbolDictionary;
+
+public class SymbolDictionarySample extends Application {

```

Comment:
Sounds good I like, I think there are a few that don't have sample at the end of their name so may need to create a small issue to update those.